website/pages/en/cookbook/subgraph-debug-forking.mdx

```diff
@@ -98,5 +98,4 @@ $ graph deploy gravity --debug-fork QmNp169tKvomnH3cPXTfGg4ZEhAHA6kEq5oy1XDqAxqH
 ```
 
 3. I inspect the logs produced by the local Graph node and, Hooray!, everything seems to be working.
-4. I deploy my now bug-free subgraph to a remote Graph node and live happily ever after! (no potatoes tho)
-5. The end...
+4. I deploy my now bug-free subgraph to a remote Graph node and live happily ever after! (no potatoes tho)
```

Comment:
```suggestion
4. I deploy my now bug-free subgraph to a remote Graph Node and live happily ever after! (no potatoes tho)
```
There are also some other "Graph node"s above that I can't make suggestions for because they are not lines that you changed.